src/codeu/chat/client/core/Controller.java

```diff
@@ -39,6 +42,13 @@ public Controller(ConnectionSource source) {
     this.source = source;
   }
 
+  public void addMemberToConvo(Uuid conversationID, Uuid owner, Uuid member) {
+    // look up conversation header based off ID
```

Comment:
In the server controller you can get the conversation by using:

`model.conversationById().first(conversationID)`